README.md

```diff
@@ -1,2 +1,25 @@
-# tsdbcg
-A Deck Building Card Game built using X and Y working with A to Z.
+## POST `api/v1/games/:id/buy_card`
+This endpont allows a player in a game to but a card for that case. A POST with the body
+```
+{
+  buy_info: {
+    player_id: id,
+    bought: {
+      "Gold" => 1,
+      "Estate" => 1
+    }
+  }
+}
+```
+
+successfully returns 
+```
+{
+  player_name: name,
+  bought: {
+    "Gold" => 1,
+    "Estate" => 1
+  }
+}
+```
+This post updates the cards in this players deck by setting the corresponding GameCard's player_id field to this player's id and discarded field to true, effectively adding it to this players discard.
```

Comment:
"This POST request will take corresponding GameCard resources in the Game and assign their `player_id` to the Players ID, and mark the `discarded` attribute as true. This will move a GameCard from the public pool of cards to the Players ownership and 'discard pile'.